src/components/Footer/Links.tsx

```diff
@@ -1,49 +1,61 @@
+import React, { ReactElement } from 'react'
 import { useUserPreferences } from '@context/UserPreferences'
 import { useGdprMetadata } from '@hooks/useGdprMetadata'
 import Button from '@shared/atoms/Button'
 import Link from 'next/link'
-import React, { Fragment } from 'react'
-import content from '../../../content/footer.json'
-import External from '@images/external.svg'
 import styles from './Links.module.css'
 import { useMarketMetadata } from '@context/MarketMetadata'
 
-export default function Links() {
-  const { appConfig } = useMarketMetadata()
+export default function Links(): ReactElement {
+  const { appConfig, siteContent } = useMarketMetadata()
   const { setShowPPC, privacyPolicySlug } = useUserPreferences()
   const cookies = useGdprMetadata()
 
+  const { content, privacyTitle } = siteContent.footer
+
   return (
-    <div className={styles.links}>
-      {content.links.map(({ name, url }) => (
-        <Fragment key={name}>
-          <Button style="text" size="small" href={url} target="_blank">
-            {name} <External />
-          </Button>
-          {' — '}
-        </Fragment>
+    <div className={styles.container}>
+      {content?.map((section, i) => (
+        <div key={i} className={styles.section}>
+          <p className={styles.title}>{section.title}</p>
+          <div className={styles.links}>
+            {section.links.map((e, i) =>
+              e.link.startsWith('/') ? (
+                <Link key={i} href={e.link}>
+                  {e.name}
+                </Link>
+              ) : (
+                <a
+                  key={i}
+                  href={e.link}
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  {e.name} &#8599;
+                </a>
```

Comment:
question: can't we use a `ExternalLink` component or smth similar here? I remember we had one in the past.